content-signup/README.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Send follow-up emails automatically
+description: Send users in an online community follow up content based on their
+interests.
+material_icon: autorenew
+create_time: 2019-07-15
+update_time: 2019-09-17
+---
+
+Contributed by Tech and Eco, follow me on [Twitter](https://twitter.com/TechandEco)
+
+By using a Google Form, let users select topics from a predefined list of
+content that you offer, and store the submissions in a Google Sheet. The
+content is then automatically sent to their email using a Google Doc as the
+email's template. The template inserts the user's name and the online content
+they selected which can be a video, whitepaper, blog, PDF summary, or other
+resources referenced via a URL.
+
+The Google Form can be accessed on your website, email signature,
+instant message, or wherever you share the form's link.
+
+![Topics are selected in a form, and upon submit an email is sent](../assets/content-signup.gif)
```

Comment:
The README preview points to `https://github.com/gsuitedevs/solutions/blob/master/assets/content-signup.gif` in master, so it should be visible once it's merged.